tests/modules/js-and-css/optimize-embeds/optimize-oembeds-test.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Tests for embed optimizer.
+ *
+ * @package performance-lab
+ * @group embed-optimizer
+ */
+
+class Embed_Helper_Tests extends WP_UnitTestCase {
+
+	/**
+	 * Test that the oEmbed HTML is filtered.
+	 *
+	 * @dataProvider test_oembed_optimize_html_data
+	 */
+	public function test_perflab_optimize_oembed_html( $html, $expected ) {
```

Comment:
This is something we do in the AMP plugin to avoid duplication.

```suggestion
	public function test_perflab_optimize_oembed_html( $html, $expected = null ) {
		if ( null === $expected ) {
			$expected = $html; // No change.
		}
```

Reply:
nice!